wiki/Beatmap_Editor/Beat_Snap_Divisor/en.md

```diff
@@ -24,14 +65,60 @@ White line: Common Time
-|--------------|--------------------------------------------|----------------------------------------------------|----------------------------------------------|-----------------------------------------------|
-| Divisor      | 1/1                                        | 1/2                                                | 1/3                                          | 1/4                                           |
-| Beat         | Full Beats                                 | Half Beats                                         | Triplets                                     | Quarter Beats                                 |
-| Ticks notes  | White ticks only                           | Adds red ticks                                     | Has two purple ticks between each white tick | Adds blue ticks, in addition to the red ticks |
-| Notes        | Best used for Easy difficulty.             |                                                    | Compound Timing                              | Useful for general mapping                    |
-| Half-measure | ![](1_6_m.jpg "1_6_m.jpg")                 | ![](1_8_m.jpg "1_8_m.jpg")                         | ![](1_12_m.jpg "1_12_m.jpg")                 | ![](1_16_m.jpg "1_16_m.jpg")                  |
-| Divisor      | 1/6                                        | 1/8                                                | 1/12                                         | 1/16                                          |
-| Beat         | Double Triplets                            | Eighth Beats                                       | Twelveth Beats                               | Sixteenth Beats                               |
-| Tick notes   | Has two purple ticks between each red tick | Adds yellow ticks, in addition to the red and blue |                                              |                                               |
-| Notes        | Compound timing                            |                                                    |                                              |                                               |
+### Divisor 1/1
+
+![](/wiki/shared/1_1_m.jpg)
+
+- Full Beats
+- White ticks only
+- Best used for Easy difficulty.
+
+### Divisor 1/2
+
+![](/wiki/shared/1_2_m.jpg)
+
+- Half Beats
+- Adds red ticks
+
+### Divisor 1/3
+
+![](/wiki/shared/1_3_m.jpg)
+
+- Triplets
+- Has two purple ticks between each white tick
+- Compound Timing
+
+### Divisor 1/4
+
+![](/wiki/shared/1_4_m.jpg)
+
+- Quarter Beats
+- Adds blue ticks, in addition to the red ticks
+- Useful for general mapping
+
+### Divisor 1/6
+
+![](/wiki/shared/1_6_m.jpg)
+
+- Double Triplets
+- Has two purple ticks between each red tick
+- Compound timing
+
+### Divisor 1/8
+
+![](/wiki/shared/1_8_m.jpg)
+
+- Eighth Beats
+- Adds yellow ticks, in addition to the red and blue
+
+### Divisor 1/12
+
+![](/wiki/shared/1_12_m.jpg)
+
+- Twelveth Beats
+
+### Divisor 1/16
+
+![](/wiki/shared/1_16_m.jpg)
```

Comment:
L122: Add text on ``[]`` to phrase which supposed to be there if the image doesn't load